diff --git a/node_modules/react-native-photo-view-ex/android/build.gradle b/node_modules/react-native-photo-view-ex/android/build.gradle
index 8b258b2..826c98a 100644
--- a/node_modules/react-native-photo-view-ex/android/build.gradle
+++ b/node_modules/react-native-photo-view-ex/android/build.gradle
@@ -58,7 +58,7 @@ repositories {
 dependencies {
     compileOnly 'com.facebook.react:react-native:+'
     implementation 'me.relex:photodraweeview:1.1.3'
-    implementation 'com.android.support:support-annotations:28.0.0'
+    implementation 'androidx.annotation:annotation:1.0.2'
 }
 
 task customClean(type: Delete) {
diff --git a/node_modules/react-native-photo-view-ex/android/src/main/java/io/amarcruz/photoview/ImageEvent.java b/node_modules/react-native-photo-view-ex/android/src/main/java/io/amarcruz/photoview/ImageEvent.java
index 23a3374..644659c 100644
--- a/node_modules/react-native-photo-view-ex/android/src/main/java/io/amarcruz/photoview/ImageEvent.java
+++ b/node_modules/react-native-photo-view-ex/android/src/main/java/io/amarcruz/photoview/ImageEvent.java
@@ -1,6 +1,6 @@
 package io.amarcruz.photoview;
 
-import androidx.support.annotation.IntDef;
+import androidx.annotation.IntDef;
 import com.facebook.react.bridge.WritableMap;
 import com.facebook.react.uimanager.events.Event;
 import com.facebook.react.uimanager.events.RCTEventEmitter;
diff --git a/node_modules/react-native-photo-view-ex/android/src/main/java/io/amarcruz/photoview/PhotoView.java b/node_modules/react-native-photo-view-ex/android/src/main/java/io/amarcruz/photoview/PhotoView.java
index f569d05..b2ac647 100644
--- a/node_modules/react-native-photo-view-ex/android/src/main/java/io/amarcruz/photoview/PhotoView.java
+++ b/node_modules/react-native-photo-view-ex/android/src/main/java/io/amarcruz/photoview/PhotoView.java
@@ -4,7 +4,7 @@ import android.content.Context;
 import android.graphics.drawable.Animatable;
 import android.graphics.drawable.Drawable;
 import android.net.Uri;
-import androidx.support.annotation.NonNull;
+import androidx.annotation.NonNull;
 
 import com.facebook.drawee.backends.pipeline.PipelineDraweeControllerBuilder;
 import com.facebook.drawee.controller.BaseControllerListener;
